doc/developer/workflow.rst

```diff
@@ -283,7 +283,9 @@ Pre-submission Checklist
    -  ``make test``
 
 - In the case of a major new feature or other significant change, document
-  plans for continued maintenance of the feature
+  plans for continued maintenance of the feature.  In addition it is a
+  requirement that automated testing must be included with the new feature
+  or significant change that is part of the normal CI system.
```

Comment:
needs to be rewritten to make it clear the meaning is that a test needs to be added, as written "included" could be interpreted to just mean "run"

Reply:
fixed